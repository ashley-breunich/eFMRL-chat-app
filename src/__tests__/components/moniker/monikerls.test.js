import React from 'react';
import renderer from 'react-test-renderer';
import MonikerLS from 'components/moniker/monikerLS.js';

describe('<Header />', () => {
  it('is alive at application start', () => {
    let app = shallow(<MonikerLS />);
    expect(app.find('div').exists()).toBeTruthy();
  });

  it('is alive at application start', () => {
    let app = shallow(<MonikerLS />);
    expect(app.find('h2').exists()).toBeTruthy();
  });

  it('renders correctly', () => {
    const header = renderer.create(<MonikerLS />).toJSON();
    expect(header).toMatchSnapshot();
  });
});
