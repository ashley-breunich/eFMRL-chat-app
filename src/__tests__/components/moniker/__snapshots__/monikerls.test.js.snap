// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MonikerLS /> Moniker renders correctly 1`] = `
<div
  className="center"
>
  <h2
    className="loginh2"
  >
    Welcome back, 
    !
  </h2>
  <button
    className="loginButton"
    value={null}
  >
    Log In
  </button>
  <h3
    className="loginh3"
  >
    Not you? Click to sign in.
  </h3>
  <button
    className="loginButton"
  >
    Sign In
  </button>
</div>
`;
